environments/staging/fab-settings.yml

```diff
@@ -1,2 +1,4 @@
 ignore_kafka_checkpoint_warning: True
 default_branch: autostaging
+py3_include_venv: False
+py3_run_deploy: False
```

Comment:
I expect to add future settings for running pillows, celery, webworkers etc in python 3.